lua/test.sh

```diff
@@ -12,7 +12,7 @@ export VIS_THEME=theme
 if [ $# -gt 0 ]; then
 	test_files=$@
 else
-	test_files=$(find . -type f -name "*.in")
+	test_files="$(find . -type f -name "*.in") basic_empty_file.in"
```

Comment:
I wanted a way to run some tests on a file that doesn't exist. Perhaps there's a better way.